chain/cosmos/gas.go

```diff
@@ -0,0 +1,38 @@
+package cosmos
+
+import (
+	"context"
+
+	"github.com/renproject/pack"
+)
+
+// A GasEstimator returns the gas-per-byte that is needed in order to confirm
+// transactions with relatively high speed. In distributed networks that work to
+// collectively build transactions, it is important that all nodes return the
+// same values from this interface.
+type GasEstimator interface {
+	GasPerByte(ctx context.Context) (pack.U64, error)
+	GasPerSignature(ctx context.Context) (pack.U64, error)
+}
```

Comment:
```suggestion
type GasEstimator interface {
	gas.Estimator // Extend the standard Gas API.
	GasPerSignature(ctx context.Context) (pack.U64, error)
}
```

Reply:
Note: you will need to add an import at the top of this file for this suggestion to be valid.